Content.IntegrationTests/Tests/Fluids/FluidSpillTest.cs

```diff
@@ -0,0 +1,174 @@
+﻿#nullable enable
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Content.Server.Fluids.Components;
+using Content.Server.Fluids.EntitySystems;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Directions;
+using Content.Shared.FixedPoint;
+using NUnit.Framework;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.Timing;
+
+namespace Content.IntegrationTests.Tests.Fluids;
+
+[TestFixture]
+[TestOf(typeof(FluidSpreaderSystem))]
+public sealed class FluidSpill : ContentIntegrationTest
+{
+    [Test]
+    public async Task SpillEvenlyTest()
+    {
+        PuddleComponent? GetPuddle(IEntityManager entManager, IMapManager mapManager1, Vector2i vector2I)
+        {
+            var grid = GetMainGrid(mapManager1);
+            foreach (var uid in grid.GetAnchoredEntities(vector2I))
+            {
+                if (entManager.TryGetComponent(uid, out PuddleComponent puddleComponent))
+                    return puddleComponent;
+            }
+
+            return null;
+        }
+
+        var server = StartServer();
+
+        await server.WaitIdleAsync();
+
+        var mapManager = server.ResolveDependency<IMapManager>();
+        var entitySystemManager = server.ResolveDependency<IEntitySystemManager>();
+        var sTileDefinitionManager = server.ResolveDependency<ITileDefinitionManager>();
+        var entityManager = server.ResolveDependency<IEntityManager>();
+        var spillSystem = entitySystemManager.GetEntitySystem<SpillableSystem>();
+        var sGameTiming = server.ResolveDependency<IGameTiming>();
+        Vector2i pos = Vector2i.One;
+        Direction[] dirs = { };
+
+        server.Post(() =>
+        {
+            dirs = SharedDirectionExtensions.RandomDirections().ToArray();
+            var solution = new Solution("Water", FixedPoint2.New(100));
+            var grid = GetMainGrid(mapManager);
+            var tileRef = GetMainTile(grid);
+            pos = tileRef.GridIndices;
+            var tileX = new Tile(sTileDefinitionManager["underplating"].TileId);
+
+            // Init 3x3 tiles
+            grid.SetTile(pos, tileX);
+            foreach (var dir in dirs)
+            {
+                grid.SetTile(pos.Offset(dir), tileX);
+            }
```

Comment:
rather than doing this you could just load the 3x3 breathing map in tests, i forget what its called exactly but its just 3x3 of tiles

Reply:
Does it have tiles?

Reply:
if it didnt the entity wouldnt be able to breathe so it should

Reply:
Tried but walls prevent spills.